plugins/txn_test_gen_plugin/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-if(WASM_TOOLCHAIN)
+if(WASM_TOOLCHAIN_FOUND)
```

Comment:
At this point, there is no "successful" cmake run where `WASM_TOOLCHAIN_FOUND` would be false.  Most of these conditionals can be removed in favor of running the side of the branch that corresponds with success.

Reply:
yes. I was trying to mimic the master branch behavior